src/nbl/video/IOpenGL_LogicalDevice.h

```diff
@@ -573,11 +572,19 @@ class IOpenGL_LogicalDevice : public ILogicalDevice, protected impl::IOpenGL_Log
             {
                 auto& p = std::get<SRequestBufferCreate2>(req.params_variant);
                 core::smart_refctd_ptr<IGPUBuffer>* pretval = reinterpret_cast<core::smart_refctd_ptr<IGPUBuffer>*>(req.pretval);
+
+                IDriverMemoryBacked::SDriverMemoryRequirements2 mreqs;
+                mreqs.size = p.creationParams.declaredSize;
+                mreqs.memoryTypeBits = 0xffffffffu;
+                mreqs.alignmentLog2 = 0u; // TODO(Erfan) Alignment previously was 0u in getXXXMemoryRequirementsOnDedMem(). what to set here? get minXXXOffsetAlignment from physical device and deduce from usage?
+                mreqs.prefersDedicatedAllocation = true;
+                mreqs.requiresDedicatedAllocation = true;
```

Comment:
I'd rather move that setup into `COpenGLBuffer` constructor, and just take declaredSize as argument